ui/apps/platform/cypress/constants/NetworkPage.js

```diff
@@ -64,6 +64,10 @@ export const selectors = {
         namespaceSelect: '.namespace-select > button',
         filterSelect: search.multiSelectInput,
     }),
+    selector: scopeSelectors('[data-testid="network-graph-selector-bar"]', {
```

Comment:
I'm now confused by the interchangeable `selector` vs. `filter` terminology. If we're calling the namespace containers in the Topology `filtered namespaces` should we not use that term for these too? We should eventually revisit this

Reply:
I agree it's confusing. I landed on "selector" for now, because of these 2 reasons.

1. We still have the search bar below it, with our traditional key-value search, which we call our "filter" bar.
2. If we updated to similar functionality using PatternFly paradigms, they refer to those as "filters".

Perhaps we coin a new term for this cluster/namespace/deployment pattern, like for instance "hierarchy" ?